test/DGmethods/compressible_Navier_Stokes/dycoms.jl

```diff
@@ -1,611 +0,0 @@
-    end
-
-    beta  = 1.0 - (1.0 - ctop) * (1.0 - csxl)*(1.0 - csxr) * (1.0 - csyl)*(1.0 - csyr)
-    beta  = min(beta, 1.0)
-    alpha = 1.0 - beta
-
-    @inbounds begin
-        S[_U] -= beta * U
-        S[_V] -= beta * V
-        S[_W] -= beta * W
-    end
-    
-end
-#---END SPONGE
-
-"""
-Geopotential source term. Gravity forcing applied to the vertical
-momentum equation
-"""
-@inline function source_geopot!(S,Q,aux,t)
-    gravity::eltype(Q) = grav
-    @inbounds begin
-        ρ, U, V, W, E  = Q[_ρ], Q[_U], Q[_V], Q[_W], Q[_E]
-        S[_V] += - ρ * gravity
-    end
-end
-
-"""
-Large scale subsidence common to several atmospheric observational
-campaigns. In the absence of a GCM to drive the flow we may need to 
-specify a large scale forcing function. 
-"""
-@inline function source_ls_subsidence!(S,Q,aux,t)
-    @inbounds begin
-        nothing
-    end
-end
-
-# ------------------------------------------------------------------
-# -------------END DEF SOURCES-------------------------------------# 
-
-# initial condition
-"""
-User-specified. Required. 
-This function specifies the initial conditions
-for the dycoms driver. 
-"""
-function dycoms!(dim, Q, t, x, y, z, _...)
-    DFloat         = eltype(Q)
-    p0::DFloat      = MSLP
-    gravity::DFloat = grav
-    
-    # ----------------------------------------------------
-    # GET DATA FROM INTERPOLATED ARRAY ONTO VECTORS
-    # This driver accepts data in 6 column format
-    # ----------------------------------------------------
-    (sounding, _, ncols) = read_sounding()
-    
-    # WARNING: Not all sounding data is formatted/scaled 
-    # the same. Care required in assigning array values
-    # height theta qv    u     v     pressure
-    zinit, tinit, qinit, uinit, vinit, pinit  = sounding[:, 1],
-    sounding[:, 2],
-    sounding[:, 3],
-    sounding[:, 4],
-    sounding[:, 5],
-    sounding[:, 6]    
-    #------------------------------------------------------
-    # GET SPLINE FUNCTION
-    #------------------------------------------------------
-    spl_tinit    = Spline1D(zinit, tinit; k=1)
-    spl_qinit    = Spline1D(zinit, qinit; k=1)
-    spl_uinit    = Spline1D(zinit, uinit; k=1)
-    spl_vinit    = Spline1D(zinit, vinit; k=1)
-    spl_pinit    = Spline1D(zinit, pinit; k=1)
-    # --------------------------------------------------
-    # INITIALISE ARRAYS FOR INTERPOLATED VALUES
-    # --------------------------------------------------
-    xvert          = y
-    
-    datat          = spl_tinit(xvert)
-    dataq          = spl_qinit(xvert)
-    datau          = spl_uinit(xvert)
-    datav          = spl_vinit(xvert)
-    datap          = spl_pinit(xvert)
-    dataq          = dataq * 1.0e-3
-    
-    randnum   = rand(1)[1] / 100
-
-    θ_liq = datat
-    q_tot = dataq + randnum * dataq
-    P     = datap    
-    T     = air_temperature_from_liquid_ice_pottemp(θ_liq, P, PhasePartition(q_tot))
-    ρ     = air_density(T, P)
```

Comment:
Not sure if this file was supposed to be deleted, but in the dycoms driver should this be
```
    ρ     = air_density(T, P, PhasePartition(q_tot))
```

Reply:
This file is now obsolete and should not be merged to the main branch.

Reply:
Same question about density applies to all the dycoms drivers I have seen